tests/functional/z_cluster/cluster_expansion/test_add_capacity_entry_exit_criteria.py

```diff
@@ -100,9 +100,10 @@ def test_add_capacity(
         # All OCS pods are in running state:
         # ToDo https://github.com/red-hat-storage/ocs-ci/issues/2361
         expected_statuses = [constants.STATUS_RUNNING, constants.STATUS_COMPLETED]
-        assert pod_helpers.check_pods_in_statuses(
+        assert pod_helpers.wait_for_pods_to_be_in_statuses(
             expected_statuses=expected_statuses,
-            exclude_pod_name_prefixes=["demo-pod"],
+            exclude_pod_name_prefixes=["demo-pod", "storageclient"],
+            timeout=500,
```

Comment:
500 sec sounds like long timeout if problem is race condition, is it possible to use shorter timeout? and what would be sampling interval?